src/wp-admin/includes/upgrade.php

```diff
@@ -2349,6 +2353,33 @@ function upgrade_640() {
 	}
 }
 
+/**
+ * Executes changes made in WordPress 6.5.0.
+ *
+ * @ignore
+ * @since 6.5.0
+ *
+ * @global int $wp_current_db_version The old (current) database version.
+ */
+function upgrade_650() {
+	global $wp_current_db_version;
+
+	if ( $wp_current_db_version < 57500 ) {
+		$update_options = array(
+			'medium_crop',
+			'medium_large_crop',
+			'large_crop',
+			'site_logo',
+		);
+
+		foreach ( $update_options as $option ) {
+			$value = get_option( $option );
+			delete_option( $option );
+			add_option( $option, $value, '', 'yes' );
+		}
```

Comment:
```suggestion
		wp_set_options_autoload( $update_options, yes );
```
In WordPress 6.4, we introduced the `wp_set_options_autoload` function to manage the autoload value for specific options. However, in certain cases, it may return early, as seen here: https://github.com/WordPress/wordpress-develop/blob/trunk/src/wp-includes/option.php#L428-L431. This occurs because these options did not have autoload values, which is why the older pattern was used to update these options. cc. @felixarntz @joemcgill

Reply:
@mukeshpanchal27 I don't think the version you're using would be appropriate. The reason `wp_set_options_autoload()` returns early is because in that case these options do not exist - in which case the code here wouldn't work either.

In fact, this approach is somewhat problematic since for non existent options it would then store them in the database as `false` (the default value for the option), which is wasteful.

More importantly though, I think the reason that you've seen issues is that as far as I can tell most of these options aren't relevant for core. See my comment in https://core.trac.wordpress.org/ticket/56912#comment:11.